### PR TITLE
Update active_record_queries.md

### PR DESCRIPTION
**Pull request by agl9**
Hi, just updated the section numbers of the rails guide mentioned as they seem to have changed.

 - [ ] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/master/CONTRIBUTING.md).

#### 1.Describe the changes made:

...your text here

#### 2. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

#XXXXX

### CODE REVIEW

File: rails_programming/advanced_forms_and_activerecord/active_record_queries.md [hunk 1]
```diff
@@ -214,14 +214,14 @@ Sometimes, you just can't get ActiveRecord to do what you want it to.  In that c
 ### Querying Basics
 1. Read the first 5 sections of the [Rails Guide on Active Record Querying](http://guides.rubyonrails.org/active_record_querying.html) for a more basic overview of query functions. Don't worry too much about batching and `#find_each`.
```
Comment (rlmoser99) [inline rails_programming/advanced_forms_and_activerecord/active_record_queries.md:215]: This should say "Read the first 6 sections..."

File: rails_programming/advanced_forms_and_activerecord/active_record_queries.md [hunk 2]
```diff
@@ -214,14 +214,14 @@ Sometimes, you just can't get ActiveRecord to do what you want it to.  In that c
...
 2. Read section 21 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#existence-of-objects) for a brief look at using `exists?`, `any?` and `many?`.
-3. Read sections 6, 7, and 21 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#group) for an understanding of aggregate functions and the calculations you can run on them.
-4. Skim sections 8-11 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#overriding-conditions).  
-5. Read section 12 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#joining-tables) to see how Rails lets you play with joining tables together.
-6. Read section 18 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#find-or-build-a-new-object) for a quick look at the helpful `find_or_create_by` methods.
+3. Read sections 6, 7, 8 and 22 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#group) for an understanding of aggregate functions and the calculations you can run on them.
```
Comment (rlmoser99) [inline rails_programming/advanced_forms_and_activerecord/active_record_queries.md:217]: This should say "Read sections 7, 8 and 22"

**Review by rlmoser99 — CHANGES_REQUESTED** [review]
Great catch to see that these numbers did not line up! Thanks for taking the time to update this lesson! After these changes are made, it should be good to merge. Thanks!

File: rails_programming/advanced_forms_and_activerecord/active_record_queries.md (hunk 2 above)
Comment (agl9) [reply rails_programming/advanced_forms_and_activerecord/active_record_queries.md]: Hi Rachel, I've made the changes. Thanks :)

**Review by rlmoser99 — APPROVED** [review]

**Pull request merged**